### PR TITLE
Doc: fix syntax = instead of :

### PR DESCRIPTION
**Pull request by Carreau**
I think that what was meant ?

### CODE REVIEW

**Comment by tlambert03** [comment]
lol, indeed.  can we get that in velin? :) hows that going btw?  pre-commit hook sometime?

**Review by tlambert03 — APPROVED** [review]

**Comment by Carreau** [comment]
> lol, indeed. can we get that in velin? :) hows that going btw? pre-commit hook sometime?

Velin development is currently on hold as it spawned a child project which is taking most of my free time, and would help with velin later on. Just going down the stack for now, and about to start unwinding soon. 

Hoping to have a prettier docs with semantic understanding of docstring, even locally in IPython(/Jupyter), which would give users (and maintainer) a compelling reason to care more about the docs, and make velin more useful:

Screen Shot 2021-01-21 at 20 19 55

Screen Shot 2021-01-21 at 20 28 45

> Legend: carefully chosen screenshot of a function for which the rendering works well, cropped to perfection to hide the things that don't work yet....

Note the fact that backticks are gone, orange is verbatim, and green is single backticks but the rendered would have infered what the thing is referring to otherwise it would be red (like typos for example), need to work on the wrapping logic to ignore escape sequence for coloring. This would free developers to have less concern of source-docstring having too many directive making things hard to read for users, and abstract over multiple docstrings format like google/numpydocs/myst format.

I've toyed with using sympay and [flatlatex](https://pypi.org/project/flatlatex/) to have :math: directived rendered as unicode as well.
and though about an integrated way to report error in docstring w/o having to install dev version, like a WYSIWYG editor for docstring to report errors. 

And that's just the tip of the iceberg.

**Pull request merged**
